src/data/equiv/fin.lean

```diff
@@ -64,62 +64,80 @@ by { cases k, refl, }
 /-- An equivalence that removes `i` and maps it to `none`.
 This is a version of `fin.pred_above` that produces `option (fin n)` instead of
```

Comment:
Is this comment still accurate with this change? (I haven't checked)

Reply:
I would say so. The idea of it is still similar to `fin.pred_above` even though its implementation no longer uses `fin.pred_above`